### PR TITLE
Reformat attachment details command to be more readable

### PR DESCRIPTION
**Pull request by chandler05**

### CODE REVIEW

**Comment by Marcono1234** [comment]
Looks fine to me (though I am not very familiar with Kotlin). Though I am not sure if keeping the additional logic for building proper English sentences (instead of just joining the names with `, `)  is really worth it.

Additionally, if I recall correctly some usernames containing special chars caused issues in the past, though I am not sure whether that applied to https://bugs.mojang.com/secure/ViewProfile.jspa?name=.

Maybe it would be easier to use a username link <code>[~<i>user</i>]</code> (see [Jira help](https://jira.atlassian.com/secure/WikiRendererHelpAction.jspa?section=links)) than to use a URL?

**Comment by chandler05** [comment]
I temporarily forgot that existed. I'll add it instead of the links

**Comment by Marcono1234** [comment]
Thanks!
I have edited my comment multiple times, could you please have a look at it again in case you have not seen the latest edits yet?

File: src/main/kotlin/io/github/mojira/arisa/modules/AttachmentModule.kt
Comment (Marcono1234) [inline src/main/kotlin/io/github/mojira/arisa/modules/AttachmentModule.kt:27]: Might be good to make it a little bit clearer what this comment describes:
```suggestion
            addRawRestrictedComment("Removed attachments:\n$commentInfo", "helper")
```

File: src/main/kotlin/io/github/mojira/arisa/modules/AttachmentModule.kt
Comment (Marcono1234) [inline src/main/kotlin/io/github/mojira/arisa/modules/AttachmentModule.kt:38]: It looks like currently the return type is `Any` because `map` returns a `List`. Additionally `subList(0, lastIndex)` would exclude the last element, wouldn't it?
What do you think about the following instead?
```kotlin
    private fun List<Attachment>.getCommentInfo() =
        map { " - [~${it.uploader!!.name}]: ${it.name}" }
            .joinToString { "\n" }
```

Comment (chandler05) [reply src/main/kotlin/io/github/mojira/arisa/modules/AttachmentModule.kt:38]: Looks like a better solution 👍

**Review by Marcono1234 — APPROVED** [review]
Looks good to me, though feel free to wait for a review from someone with more Kotlin experience.

**Review by urielsalis — APPROVED** [review]

**Pull request merged**
